src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const accessibilityQuiz = [
+
+  {
+    Question: "What does WCAG (sometimes pronounced wee-kag) stand for?",
```

Comment:
This question is redundant since it already exists

Reply:
I am sorry i did not see that it already exists , can I add another one?

Reply:
@Rishav9911 

Yes, you can come up with another question if you like

Reply:
Certainly! Feel free to proceed, just ensure there are no syntax errors.